paasta_tools/cli/cmds/secret.py

```diff
@@ -341,7 +342,10 @@ def is_service_folder(soa_dir, service_name):
 
 
 def _get_secret_provider_for_service(
-    service_name, cluster_names=None, soa_dir=None, secret_provider_extra_kwargs=None
+    service_name,
+    cluster_names=None,
+    soa_dir=None,
+    secret_provider_extra_kwargs=None,
 ):
```

Comment:
```suggestion
    service_name: str,
    cluster_names: Optional[str]=None,
    soa_dir: Optional[str]=None,
    secret_provider_extra_kwargs: Optional[Dict[str, str]=None,
) -> SecretProvider:
```
would be cool (although probably overkill) to have a non-total TypedDict for the kwargs we're passing here unless `get_secret_provider` and it's callees can handle any arbitrary kwarg that's passed in (in which case, nvm :p)

that said: looks like all we did here was reformat, so feel free to drop this comment